packages/ui/src/internalComponents/Popup/index.tsx

```diff
@@ -67,7 +67,7 @@ const StyledTooltip = styled('div', {
   text-align: center;
   position: absolute;
   max-width: ${({ maxWidth }) => maxWidth}px;
-  word-break: break-all;
+  overflow-wrap: anywhere;
```

Comment:
```suggestion
  overflow-wrap: break-word;
```

this is not this one ?

Reply:
it's because inside a tooltip we don't want that our word break ?